docs/developer-guide/plugins.md

```diff
@@ -311,14 +314,6 @@ module.exports = stylelint.createPlugin(ruleName, (expectation) => {
 
 If your plugin can accept an array as its primary option, you must designate this by setting the property `primaryOptionArray = true` on your rule function. For more information, check out the ["Working on rules"](rules.md) doc.
 
-## External helper modules
-
-In addition to the standard parsers mentioned in the ["Working on rules"](rules.md) doc, there are other external modules used within Stylelint that we recommend using. These include:
-
-- [postcss-resolve-nested-selector](https://github.com/davidtheclark/postcss-resolve-nested-selector): given a (nested) selector in a PostCSS AST, return an array of resolved selectors.
```

Comment:
Removed so we don't advocate using such an old package.

Added "construct-specific parsers" to the list at the top of built-in rule conventions to follow.